extensions/blocks/podcast-episodes/edit.js

```diff
@@ -3,49 +3,126 @@
  */
+import { Component } from '@wordpress/element';
+import {
+	Button,
+	Disabled,
+	ExternalLink,
+	PanelBody,
+	Placeholder,
+	RangeControl,
+	TextControl,
+	Toolbar,
+} from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { BlockIcon } from '@wordpress/block-editor';
-import { Placeholder, withNotices } from '@wordpress/components';
-import { useState } from '@wordpress/element';
+import { BlockControls, BlockIcon, InspectorControls } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
-import icon from './icon';
-
-function PodcastEpisodesEdit( {
-	attributes,
-	className,
-	noticeOperations,
-	noticeUI,
-	setAttributes,
-} ) {
-	/**
-	 * Write the block editor UI.
-	 *
-	 * @returns {object} The UI displayed when user edits this block.
-	 */
-	const [ notice, setNotice ] = useState();
-
-	/* Call this function when you want to show an error in the placeholder. */
-	const setErrorNotice = () => {
-		noticeOperations.removeAllNotices();
-		noticeOperations.createErrorNotice( __( 'Put error message here.', 'jetpack' ) );
-	};
-
-	return (
-		<div className={ className }>
-			<Placeholder
-				label={ __( 'Podcast Episodes', 'jetpack' ) }
-				instructions={ __( 'Instructions go here.', 'jetpack' ) }
-				icon={ <BlockIcon icon={ icon } /> }
-				notices={ noticeUI }
-			>
-				{ __( 'User input goes here?', 'jetpack' ) }
-			</Placeholder>
-		</div>
-	);
+import { edit, queueMusic } from './icons/';
+
+const DEFAULT_MIN_ITEMS = 1;
+const DEFAULT_MAX_ITEMS = 10;
+
+class PodcastEpisodesEdit extends Component {
+	constructor() {
+		super( ...arguments );
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+		this.onSubmitURL = this.onSubmitURL.bind( this );
+	}
+
+	toggleAttribute( propName ) {
+		return () => {
+			const value = this.props.attributes[ propName ];
+			const { setAttributes } = this.props;
+
+			setAttributes( { [ propName ]: ! value } );
+		};
+	}
+
+	onSubmitURL( event ) {
+		event.preventDefault();
+	}
+
+	render() {
+		const { url, itemsToShow } = this.props.attributes;
+		const { attributes, setAttributes } = this.props;
+
+		if ( ! url ) {
+			return (
+				<Placeholder
+					icon={ <BlockIcon icon={ queueMusic } /> }
+					label={ __( 'Podcast Episodes', 'jetpack' ) }
+					instructions={ __( 'Paste a link to your Podcast RSS feed.', 'jetpack' ) }
+				>
+					<form onSubmit={ this.onSubmitURL }>
+						<TextControl
+							placeholder={ __( 'Enter URL here…', 'jetpack' ) }
+							value={ url || '' }
+							onChange={ value => setAttributes( { url: value } ) }
+							className={ 'components-placeholder__input' }
+						/>
+						<Button isSecondary type="submit">
+							{ __( 'Embed', 'jetpack' ) }
+						</Button>
+					</form>
+					<div className="components-placeholder__learn-more">
+						<ExternalLink href={ __( 'https://wordpress.org/support/article/embeds/' ) }>
```

Comment:
Here we'll most likely need 2 support docs, one for wpcom and one for Jetpack. See how it's done for other blocks here:
https://github.com/Automattic/jetpack/blob/a129d3f4fddb692531cc9fceb0e04310d14a26bb/extensions/blocks/eventbrite/edit.js#L259-L262

Reply:
I'm not entirely sure we have those docs yet. What would you suggest now? Comment out the link and add it back once we have docs ready?

Reply:
I think it's fine to create empty drafts for each doc, and use the links in the block even though they point to drafts for now.

Reply:
Can somebody please update URLs?

Dotcom: https://en.support.wordpress.com/?page_id=163160
Jetpack: https://jetpack.com/?post_type=jetpack_support&p=95361

I don't know if we can use "nice URLs" (without ids in them) when those docs are only drafts for now.

Reply:
> I don't know if we can use "nice URLs" 

I think you can:

- https://jetpack.com/support/jetpack-blocks/podcast-player-block/
- http://en.support.wordpress.com/wordpress-editor/blocks/podcast-player-block/

Reply:
Will update.

Reply:
Fixed.